keps/sig-node/runtime-class.md

```diff
@@ -270,6 +271,34 @@ the Pod. The admission check on a replica recreation will prevent the scheduler
 the `RuntimeHandler` is not recognized by the CRI implementation, then `RunPodSandbox` will return
 an error.
 
+#### Monitoring
+
+The first round of monitoring implementation for `RuntimeClass` will cover the
+following three areas:
+
+- `how popular every runtime is?` This can be done by adding a new label into
+  [kubelet\_running\_pod\_count](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/metrics/metrics.go#L256)
+  to track the RuntimeClass information.
+- `how robust every runtime is?` This can be done by adding a new metric to
+  track the errors of different RuntimeClass of
+  [RunPodSandbox](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/instrumented_services.go#L179)
+  operations.
+- `how expensive every runtime is in terms of time overhead?` This can be done
+  by adding a new metric to track the latencies of different RuntimeClass of
+  [RunPodSandbox](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kuberuntime/instrumented_services.go#L179)
+  operations.
+
+The following monitoring areas will be skipped for now, and be considered after
+the RuntimeClass scheduling is implemented:
+
+- how many runtimes a cluster supports?
+- how many scheduling failures caused by unsupported runtimes or insufficient
+  resources of a certain runtime?
+
+Currently, we assume that all the nodes in a cluster are homogeneous. After
+heterogeneous clusters are implemented, we may need to monitor how many runtimes
+a node supports.
+
 [runpodsandbox]: https://github.com/kubernetes/kubernetes/blob/b05a61e299777c2030fbcf27a396aff21b35f01b/pkg/kubelet/apis/cri/runtime/v1alpha2/api.proto#L344
 
 ### Risks and Mitigations
```

Comment:
Can you delete `Details TBD.` on [line 361](https://github.com/kubernetes/enhancements/pull/756/files#diff-d9093bb5b949f3a78d08184362eb3d48R361)